lib/game.dart

```diff
@@ -98,7 +98,7 @@ class GameClass extends Forge2DGame
   }
 
   void _onLost() {
-    overlays.add('LostDialog');
+    overlays.isActive('WonDialog') ? null : overlays.add('LostDialog');
```

Comment:
it's fine but I would probably do
`if(!overlays.isActive(...`

Reply:
Noted, have changed